src/OrbitGl/Batcher.cpp

```diff
@@ -266,6 +266,9 @@ const PickingUserData* Batcher::GetUserData(PickingId id) const {
       return user_data_[id.element_id].get();
     case PickingType::kPickable:
       return nullptr;
+    case PickingType::kCount:
+      UNREACHABLE();
+      return nullptr;
```

Comment:
either UNREACHABLE() or return nullptr; I do not think you can do both

Reply:
I thought I'd get an error saying that not all paths return a value, I don't. I removed the "return nullptr".